packages/@react-aria/interactions/src/useHover.ts

```diff
@@ -124,13 +128,15 @@ export function useHover(props: HoverProps): HoverResult {
     };
 
     let triggerHoverEnd = (event, pointerType) => {
+      state.pointerType = '';
+      state.target = null;
+      
       if (pointerType === 'touch' || !state.isHovered) {
         return;
       }
```

Comment:
Just wanna make sure I understand the reasoning here: 
I was wondering if we should instead early return if isDisabled is true, but I guess its a good idea to still fire `onHoverEnd` and `onHoverChange` when an element gets disabled while being hovered so that the user properly respond to this specifically scenario (e.g. if they have other bits of controlled state that they need to update via these callbacks)

Reply:
Right, this is to enable cleanup of everything, otherwise if they have a style set when hovered, it'll never clear out, which was actually part of the bug this was written to address